cli/ocsprefresh/ocsprefresh.go

```diff
@@ -108,5 +139,40 @@ func SignerFromConfig(c cli.Config) (ocsp.Signer, error) {
 	return ocsp.NewSignerFromFile(c.CAFile, c.ResponderFile, k, time.Duration(c.Interval))
 }
 
+// serializeSCTRecords converts a slice of certdb.SCTRecords into an ASN.1
+// encoded SCT list.
+func serializeSCTRecords(sctRecords []certdb.SCTRecord) ([]byte, error) {
+	var scts []ct.SignedCertificateTimestamp
+
+	for _, sctRecord := range sctRecords {
+		serializedSCT, err := hex.DecodeString(sctRecord.Body)
+		if err != nil {
+			return nil, cferr.Wrap(cferr.CTError, cferr.Unknown,
```

Comment:
Rather than cferr.Unknown, it might be useful to have encoding and decoding reasons in cferr.